src/applications/mhv-medical-records/containers/LandingPage.jsx

```diff
@@ -43,6 +47,23 @@ const LandingPage = () => {
     state => state.featureToggles.mhv_integration_medical_records_to_phase_1,
   );
 
+  const useAcceleratedApi = useSelector(
```

Comment:
We've got this logic in 3 different places, would it be worth extracting it into a util or custom hook?

Reply:
1000%. Thats part of the refactor/hook work that is in the canvas

Reply:
Added the hook while waiting for tests to run